data/metric/metrics.go

```diff
@@ -25,48 +32,75 @@ const (
 	ATTRIBUTE SourceType = iota
 )
 
+const (
+	// NSSeparator is the metric namespace separator
+	NSSeparator = "::"
+	// NSAttributeSeparator is the metric attribute key-value separator applied to generate the metric ns.
+	NSAttributeSeparator = "=="
+)
+
+// Errors
+var (
+	ErrNonNumeric        = errors.New("non-numeric value for rate/delta")
+	ErrNoStoreToCalcDiff = errors.New("can't use deltas nor rates without persistent store")
```

Comment:
Avoid using contractions on error messages. You can use `cannot` instead.

Reply:
what's the gain on switching `ErrNoStoreToCalcDiff`to `ErrCannotCalcDiffWithoutStore`?

Reply:
I'm talking about the error message.

Reply:
ops, sure 👍